setup.py

```diff
@@ -66,6 +66,9 @@
             'require = sifter.commands.require:CommandRequire',
             'set = sifter.commands.variables:CommandSet',
             'stop = sifter.commands.stop:CommandStop',
+            'imap4flags.setflag = sifter.commands.imap4flags:CommandSetFlag',
```

Comment:
@manfred-kaiser i'm not 100% on the best way to declare these. I put a dot in the name because there are 3 commands to the imap4flags

Reply:
i think these names are just an annotation and aren't used in the logic